### PR TITLE
StakeValidator.CheckStakeKernelHash redundant variables removal

### PR DESCRIPTION
**Pull request by noescape00**
Removed redundant input variables from StakeValidator.CheckStakeKernelHash and updated the entry log

### CODE REVIEW

File: src/Stratis.Bitcoin.Features.Consensus/StakeValidator.cs [hunk 1]
```diff
@@ -362,13 +353,12 @@ private bool VerifySignature(UnspentOutputs coin, Transaction txTo, int txToInN,
         /// </remarks>
         /// <exception cref="ConsensusErrors.StakeTimeViolation">Thrown in case transaction time is lower than it's own UTXO timestamp.</exception>
         /// <exception cref="ConsensusErrors.StakeHashInvalidTarget">Thrown in case PoS hash doesn't meet target protocol.</exception>
-        private void CheckStakeKernelHash(ContextStakeInformation context, ChainedBlock prevChainedBlock, uint headerBits, uint prevBlockTime,
-            BlockStake prevBlockStake, UnspentOutputs stakingCoins, OutPoint prevout, uint transactionTime)
+        private void CheckStakeKernelHash(ContextStakeInformation context, uint headerBits, BlockStake prevBlockStake, UnspentOutputs stakingCoins,
```
Comment (dangershony) [inline src/Stratis.Bitcoin.Features.Consensus/StakeValidator.cs]: What all those where not used?
Hmm probably leftover of the V2 deleted code

Comment (noescape00) [reply src/Stratis.Bitcoin.Features.Consensus/StakeValidator.cs]: >What all those where not used?

Yeah. 

>Hmm probably leftover of the V2 deleted code

So did @Aprogiena said. He was refactoring this class before.

File: src/Stratis.Bitcoin.Features.Consensus/StakeValidator.cs [hunk 2]
```diff
@@ -162,13 +162,6 @@ public void CheckProofOfStake(ContextStakeInformation context, ChainedBlock prev
             if ((coins == null) || (coins.UnspentOutputs.Length != 1))
                 ConsensusErrors.ReadTxPrevFailed.Throw();
 
-            ChainedBlock prevBlock = this.chain.GetBlock(coins.BlockHash);
```
[the remark this answers is not in the document]
Comment (noescape00) [reply src/Stratis.Bitcoin.Features.Consensus/StakeValidator.cs]: Yes, that was my way of thinking. 
StakeValidator is probably not the component that should handle a reorg.

[the remark this answers is not in the document]
Comment (Aprogiena) [reply src/Stratis.Bitcoin.Features.Consensus/StakeValidator.cs]: i'd remove it

[the remark this answers is not in the document]
Comment (dangershony) [reply src/Stratis.Bitcoin.Features.Consensus/StakeValidator.cs]: Cool yeah so we all agree.

**Pull request merged**